tools/xmlGenerator/domainGenerator.py

```diff
@@ -209,7 +209,9 @@ def main():
                             args.schemas_dir],
                            stdout=sys.stdout, stdin=subprocess.PIPE, stderr=sys.stderr)
 
-    initial_settings = os.path.realpath(args.initial_settings)
+    initial_settings = ""
```

Comment:
|> ""| evaluates to |False|,
None too, so I do not see the drawback of using `None` instead of `""`.
---------------------------------------------------------------------
Intel Corporation SAS (French simplified joint stock company)
Registered headquarters: "Les Montalets"- 2, rue de Paris, 
92196 Meudon Cedex, France
Registration Number:  302 456 199 R.C.S. NANTERRE
Capital: 4,572,000 Euros

This e-mail and any attachments may contain confidential material for
the sole use of the intended recipient(s). Any review or distribution
by others is strictly prohibited. If you are not the intended
recipient, please contact the sender and delete all copies.